Library/Formula/nginx.rb

```diff
@@ -121,12 +120,7 @@ def post_install
     end
   end
 
-  test do
-    system "#{bin}/nginx", "-t"
-  end
-
   def passenger_caveats; <<-EOS.undent
-
```

Comment:
This line is actually deliberate. Without it the two caveats will rub up against each other directly when both apply. The empty line ensures there's a gap between them. I'll pull it and add the line back though, no point making the bot do it again.